keps/sig-scheduling/4816-dra-prioritized-list/README.md

```diff
@@ -396,17 +396,17 @@ type DeviceRequest struct {
     // +oneOf=deviceRequestType
     Exactly *SpecificDeviceRequest
 
-    // FirstAvailable specified a list of DeviceSubRequests, at least
-    // one of which must be satisfied in order to satisfy this request.
-    // The scheduler will attempt to satisfy the list of subrequests in
-    // order, and it will select the first one that satisfies the
-    // requirements of the request. So if there are two entries in the
-    // list, the schduler will only check the second one if it determines
-    // that the first one can not be used.
+    // FirstAvailable contains subrequests, of which exactly one will be 
+    // satisfied by the scheduler to satisfy this request. It tries to
+    // satisfy them in the order in which they are listed here. So if
+    // there are two entries in the list, the schduler will only check
+    // the second one if it determines that the first one can not be used.
     //
     // DRA does not yet implement scoring, so the scheduler will
     // select the first set of devices that satisfies all the
-    // requests in the claim. This means that the set of devices allocated
+    // requests in the claim. And if the requirements can
+    // be satisfied on more than one node, a random node
```

Comment:
It's not fully random. It depends on what other plugins say. But, arguably, it's hard to predict.

I would say something like "other scheduling features will determine which node is chosen"

Reply:
Updated.